frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx

```diff
@@ -136,6 +143,17 @@ const SettingsForm: React.FC = () => {
             <SubmitButton isChange={isChange()}>
                 Update My Settings
             </SubmitButton>
+            <div className={styles.dltButton}>
+                <Button variant={ButtonVariant.DELETE} size={ButtonSize.MEDIUM}>
+                    <FontAwesomeIcon
```

Comment:
Don't use FontAwesomeIcon directly. Move it to a separate component. 
You can discuss the creation of this component with @ValeriiArt, as he has the same comments

Reply:
fixed